src/adaptors/connectElasticsearch.ts

```diff
@@ -0,0 +1,2 @@
+import { Client } from '@elastic/elasticsearch'
+export default new Client({ node: 'http://localhost:9200' })
```

Comment:
Move this setting to `.env` file. For example:
`ES_NODE_URL='http://localhost:9200'`